lib/ast/function_def.js

```diff
@@ -190,6 +190,17 @@ class ExpressionSignature {
     getArgType(arg) {
         return this._argmap[arg] ? this._argmap[arg].type : undefined;
     }
+    getArgCanonical(arg) {
+        const a =this._argmap[arg];
+        if (a) {
+            if (typeof a.canonical === 'string')
+                return a.canonical;
+            if (typeof a === 'object' && 'npp' in a)
+                return a['npp'];
+        }
+        return arg;
+    }
```

Comment:
Would this be better as `.canonical` or `.getCanonical` on the argument object?
Also, I thought we did not want to hardcode `npp` because that's English-specific?